libs/journeys/ui/src/components/StepFooter/FooterButtonList/ReactionButton/ReactionButton.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import { Story, Meta } from '@storybook/react'
+import { ComponentProps } from 'react'
+import Stack from '@mui/material/Stack'
+import Typography from '@mui/material/Typography'
+import { simpleComponentConfig } from '../../../../libs/simpleComponentConfig'
+import { ReactionButton } from './ReactionButton'
+
+const ReactionButtonDemo = {
+  ...simpleComponentConfig,
+  component: ReactionButton,
+  title: 'Journeys-Ui/StepFooter/FooterButtonList/ReactionButton'
+}
+
+const Template: Story<ComponentProps<typeof ReactionButton>> = () => (
+  <Stack
+    direction="row"
+    gap={12}
+    sx={{ pb: 2, width: '584px' }}
+    justifyContent="start"
+  >
+    <Stack direction="column" gap={4} justifyContent="center">
+      <Typography>Like Icon</Typography>
```

Comment:
We can use like/dislike, but is it cool if we keep the naming the same across the component?
```suggestion
      <Typography>ThumbsUp Icon</Typography>
```